frontends/web/src/utils/subscriptions.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright 2021 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Unsubscribe } from './event';
+
+export type UnsubscribeList = Unsubscribe[];
+
+export function unsubscribe(subscribtions: UnsubscribeList) {
```

Comment:
Would really appreciate some comments. Why does an `unsubscribe` function calls `subscription()`, as if it calls every other subscriber during unsubscribe? I'm just very confused here.

Reply:
Sorry, my PR was a bit too early, I'll add comments.

Reply:
Renamed the variable to make it more clear, fixed typo and added a comment